drivers/xen/xenapi/Config.uk

```diff
@@ -0,0 +1,5 @@
+config LIBXENAPI
```

Comment:
I would rename it to LIBXEN similar to another projects, such as https://github.com/xen-troops/zephyr-xenlib.

Reply:
Hum, Zephyr's Xenlib looks to me like a complete platform library and thus richer than just a place to maintain the copy of official Xen headers retrieved from: [xen.git/xen/include/public](https://xenbits.xen.org/gitweb/?p=xen.git;a=tree;f=xen/include/public;h=316fe2841855015f6da322d37513569777b41b24;hb=bd51e573a730efc569646379cd59ccba967cde97). I was also original thinking to call it `LIBXENHEADERS`.

Reply:
IMO `LIBXENHEADERS` is clearer.

Reply:
I am renaming 👍🏻